src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,192 @@
+# Data Entry (beta) app
+
+## About the Data Entry (beta) app { #aggregate_data_entry_app.about }
+
+The Data Entry (beta) app is used to enter aggregate data in DHIS2. Aggregate data is collected at a group level and doesn't belong to any one individual. Looking to enter individual-level data? Check out the [Tracker Capture app](#tracker_capture_app).   
+
+## What makes a data entry form? { #aggregate_data_entry_app.what_makes_a_form }
+
+Data entry forms are made from different choices, which make up the *context*:
```

Comment:
Since we never refer to the word `context` again in the documentation, I wonder if we might not include it? It sounds a bit technical?

Reply:
True. I remember it jumped out at me a bit when I first read it. I think it would be a little softer if it wasn't in italics.
We could also have an alternative wording?

```suggestion
Data entry forms are filled in for a specific context, which is defined by the following choices:
```

Reply:
Yeah agreed with these points, I think @Philip-Larsen-Donnelly's suggestion is an elegant way to include it and make it not-too-technical.